File: java/src/test/java/cucumber/agile2010/CustomerSteps.java
package cucumber.agile2010;

import cuke4duke.annotation.I18n.EN.Given;
import cuke4duke.annotation.I18n.EN.When;
import cuke4duke.annotation.I18n.EN.Then;

import static org.junit.Assert.assertTrue;

public class CustomerSteps {
    private CustomerList customers;

    @Given("^there are no customers in the system$")
    public void noCustomersInTheSystem() {
        customers = new CustomerList();
    }
}

File: dotnet/Agile2010/CustomerSteps.cs
using System;
using System.Collections.Generic;
using Cuke4Nuke.Framework;

namespace Agile2010
{
    public class CustomerSteps
    {
        private CustomerList customers;

        [Given(@"^there are no customers in the system$")]
        public void ThereAreNoCustomersInTheSystem()
        {
            customers = new CustomerList();
        }
    }
}
